assets/src/dashboard/components/cardGridItem/cardTitle.js

```diff
@@ -85,17 +85,28 @@ const CardTitle = ({
     if (!displayDate) {
       return null;
     }
-    return status === STORY_STATUS.PUBLISHED
-      ? sprintf(
-          /* translators: %s: last modified date */
+
+    switch (status) {
+      case STORY_STATUS.PUBLISH:
+        return sprintf(
+          /* translators: %s: published date */
```

Comment:
This is not accurate either.

Right now `displayDate` is the date the post was last modified, not the published date.

Example:

I just published a story yesterday:

<img width="601" alt="Screenshot 2020-07-09 at 10 16 14" src="https://user-images.githubusercontent.com/841956/87015167-56fb7d00-c1cd-11ea-876a-6897e670bc6a.png">

Today I fixed a typo and click "Update". The last modified date is changed, but the publish date stays the same. In the dashboard it looks like this:

<img width="477" alt="Screenshot 2020-07-09 at 10 16 22" src="https://user-images.githubusercontent.com/841956/87015241-6f6b9780-c1cd-11ea-8bc1-ddab30964656.png">

In the list view it's not an issue because we have both dates there, but in the grid view we need to use the actual publish date for published stories.